res/skins/Tango/deck_row_transport_right.xml

```diff
@@ -18,6 +18,19 @@ Variables:
     <Layout>horizontal</Layout>
     <Children>
 
+      <Template src="skin:button_2state_right_display.xml">
+        <SetVariable name="TooltipId">cue_default_cue_gotoandstop</SetVariable>
+        <SetVariable name="ObjectName">CueButton</SetVariable>
+        <SetVariable name="Size">24f,24f</SetVariable>
+        <SetVariable name="state_0_text">Q</SetVariable>
+        <SetVariable name="state_1_text">Q</SetVariable>
+        <SetVariable name="ConfigKey"><Variable name="group"/>,cue_default</SetVariable>
+        <SetVariable name="ConfigKeyRight"><Variable name="group"/>,cue_gotoandstop</SetVariable>
+        <SetVariable name="ConfigKeyDisp"><Variable name="group"/>,cue_indicator</SetVariable>
+      </Template>
+
+      <WidgetGroup><ObjectName>Spacer0f</ObjectName><Size>1f,1min</Size></WidgetGroup>
+
```

Comment:
IIRC the cue button is in between the play button and th ehotcues on Tango right decks, so thi sneeds to be moved after the Play button